docs/solutions/hypervisors/kvm/kvm-baremetal/modules/kvm-installer-baremetal/README.md

```diff
@@ -0,0 +1,128 @@
+KVM Installation module
```

Comment:
The baremetal and nested examples are very similar. Not blocking, but might make sense to have a README one level up that has some of the common info and explains the difference.

Reply:
Agreed. I just created another README as you suggested.